### PR TITLE
feat: add bazzite media mentions

### PR DESCRIPTION
**Pull request by castrojo**
BAZZITE HAS ARRIVED.

### CODE REVIEW

File: docs/media.md [hunk 1]
```diff
@@ -14,8 +16,28 @@
...
 
+### Bazzite
+
+<div class="grid" markdown>
+**Gaming on Linux**: [Bazzite is a new Fedora-based Linux for Steam Deck and gaming PCs](https://www.gamingonlinux.com/2023/08/bazzite-is-a-new-fedora-based-linux-for-steam-deck-and-gaming-pcs/)
+{ .card }
+</div>
+
+<div class="grid" markdown>
+**Techspot**: [Linux desktop users can get the Steam Deck experience with Bazzite]([https://www.gamingonlinux.com/2023/08/bazzite-is-a-new-fedora-based-linux-for-steam-deck-and-gaming-pcs/](https://www.techspot.com/news/99902-linux-desktop-users-can-get-steam-deck-experience.html))
```
[the remark this answers is not in the document]
Comment (p5) [reply docs/media.md:27]: In fact, I think this is meant to be 
```
**Techspot**: [Linux desktop users can get the Steam Deck experience with Bazzite](https://www.techspot.com/news/99902-linux-desktop-users-can-get-steam-deck-experience.html)
```
?

Comment (p5) [inline docs/media.md:27]:
```suggestion
**Techspot**: [Linux desktop users can get the Steam Deck experience with Bazzite](https://www.techspot.com/news/99902-linux-desktop-users-can-get-steam-deck-experience.html)
```

**Review by p5 — APPROVED** [review]
Don't think approving will do anything without contributor, but it's the effort that counts :)

**Review by storyaddict — CHANGES_REQUESTED** [review]
Idk why, but when I run this locally the last YouTube video is shown twice, despite the URLs being different.

**Review by storyaddict — APPROVED** [review]
Duplicate video seems to have been a cache issue. Resolved after clearing cache. Approved.

**Pull request merged**